sdk/cosmos/azure-cosmos-spark_3_2-12/src/main/scala/com/azure/cosmos/spark/CosmosConfig.scala

```diff
@@ -812,7 +812,10 @@ private object SerializationDateTimeConversionModes extends Enumeration {
   type SerializationDateTimeConversionMode = Value
 
   val Default: SerializationDateTimeConversionModes.Value = Value("Default")
-  val AlwaysEpochMilliseconds: SerializationDateTimeConversionModes.Value = Value("AlwaysEpochMilliseconds")
+  val AlwaysEpochMillisecondsWithUtcTimezone:
+    SerializationDateTimeConversionModes.Value = Value("AlwaysEpochMilliseconds")
+  val AlwaysEpochMillisecondsWithSystemDefaultTimezone:
```

Comment:
what if there is a customer want default+SystemDefaultTimeZone (is that possible?)

Reply:
Would happen already - no explicit time zone conversion happening in that code path - see CosmosRowConverter.scala